ash/src/vk/native.rs

```diff
@@ -99,6 +99,8 @@ pub const StdVideoH264ChromaFormatIdc_STD_VIDEO_H264_CHROMA_FORMAT_IDC_444:
     StdVideoH264ChromaFormatIdc = 3;
 pub const StdVideoH264ChromaFormatIdc_STD_VIDEO_H264_CHROMA_FORMAT_IDC_INVALID:
     StdVideoH264ChromaFormatIdc = 2147483647;
+pub const StdVideoH264ChromaFormatIdc_STD_VIDEO_H264_CHROMA_FORMAT_IDC_MAX_ENUM:
+    StdVideoH264ChromaFormatIdc = 2147483647;
```

Comment:
Should we be emitting MAX_ENUM values?

Reply:
Probably not, but I'm also not sure if we should go to the length of removing these from `bindgen` output. The regex for this should be just `\w+_MAX_ENUM` in `blocklist-item`, I'll give it a go.

Reply:
Ah, didn't realize this was the bindgen output! No worries if it's a pain.

Reply:
I thought this was trivial with `.blocklist_item(r"\w+_MAX_ENUM")` but apparently (at least back in 2019, haven't found anything recent) this requires a custom callback: https://github.com/rust-lang/rust-bindgen/issues/1526

Bit ugly, but it works.